shared/javascripts/options.js

```diff
@@ -63,6 +63,16 @@ if (Privly === undefined) {
   }
   Privly.options = {};
 
+  // CommonJS Module
+  if (typeof module !== "undefined" && module.exports) {
+    module.exports.options = Privly.options;
+    // load dependencies
+    ls = require("./local_storage.js").ls;
+    Privly.storage = require("./storage.js").storage;
```

Comment:
Well, there won't be anything already defined for a module. A CommonJS module can be thought of a black box, sort of separated from the outside env, or already loaded scripts. Whatever the black box needs, it asks for, and resolves the dependencies.

But there's still a certain level of optimization that can be done. The current way to define the Privly namespace in the background script is  this [file](https://github.com/hitesh96db/privly-jetpack/blob/options_interface/lib/privly_ns.js). But we know that `glyph.js` loads `options.js`, and `options.js` loads `storage.js` and `context_messenger.js`. So, we can make `options.js` export the loaded `storage.js` and `context_messenger.js`, all three of which now becomes available to `glyph.js` and then `glyph.js` can export all three and itself to the background script. So , [this](https://github.com/hitesh96db/privly-jetpack/blob/options_interface/lib/privly_ns.js) file which requires each of them separately can now require all of them using only `glyph.js`. 
I hope that made some sense ?

So we can optimize it at the background script but I doubt we can do anything about the individual modules. It's still a step forward if this works out :)